src/interactive/workspace.ts

```diff
@@ -21,7 +21,7 @@ const requestTypeGetVariables = new rpc.RequestType<
     void, void>('repl/getvariables')
 
 const requestTypeGetLazy = new rpc.RequestType<
-    void,
+    number,
```

Comment:
@pfitzseb I was not 100% certain, but it seemed this should be number? If so, then this was just an unrelated bug that was also fixed :)

Reply:
Huh, indeed. Weird that TS didn't complain about this.

Reply:
That's kinda expected, since `id` filed in `WorkspaceVariable` is type annotated as `any`:
https://github.com/julia-vscode/julia-vscode/blob/5be700c8256e5e2cf66e5f93a131fcecc2f1b096/src/interactive/workspace.ts#L11

So if we change this to `number`, it would make sense to change that as well.

Reply:
Ok, no idea why I did that :)